src/MDMPerforming.h

```diff
@@ -114,3 +114,31 @@ NS_SWIFT_NAME(ComposablePerforming)
     NS_SWIFT_NAME(set(transactionEmitter:));
 
 @end
+
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+@optional
```

Comment:
Sure thing. Makes sense. I guess if people aren't interested in implementing both of these methods, then `NamedPlanPerforming` isn't really for them.